### ISSUE TITLE
[Screen reader -Howdyai]: Narrator is reading the incorrect information for Howdy link present in the howdy page. 

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#7]
**User Experience**
If screen reader is not descriptive for the controls, screen reader user will face difficulty in understand  the type of the control.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://howdy.ai/
Browser: Edge
Screen reader: Narrator 

**Repro Steps**
Turn on the narrator 
Steps to Follow Install Slack:
Step 1: Open the URL: https://howdy.ai/
Step 2: Navigate to "@howdy for Slack" and select it.
Step 3: Navigate to "Create Workspace" and select it.
Step 4: Sign In with valid credentials to open slack account.
Step 5: Now go to Howdy Home page URL: https://howdy.ai/
Step 6: Repeat Step 2 again and select it.
Step 7: Navigate to "Install" button and select it.
Step 8: Select "Authorize" button and select it.
Step 9: Select "Turn your Boot On"
Step 10: Click on "Continue" button.
Step 11: "Howdy Dashboard" page gets opened.
Step 12: Navigate to "Launch Howdy in Slack".
Step 13: Navigate to "Dashboard" tab and select it.
Step 14: "Howdy Dashboard" page gets opened.
Step 15: Navigate to “Howdy” control 
Step 16: Observe the issue.

**Actual Result**
Narrator is reading as “Banner link “ for Howdy link which is present in the howdy page.

**Expected Result** 
Narrator should read as “Howdy home page back to dashboard” link.

Note: In scan mode narrator is reading as unnecessary info for Howdy link 
Please refer the attachment: MAS1.3.1_Scan mode

Note1: Help link narrator is reading as Menu help link.
Please refer the attachment:MAS1.3.1_Help link.

Note 2:Narrator is not reading the state for Pause all button when the user activate it.
Please refer the attachment:MAS1.3.1_Pause button.wmv

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
[MAS1.3.1_Help link.zip](https://github.com/howdyai/howdy-website/files/2973576/MAS1.3.1_Help.link.zip)

[MAS1.3.1_Howdy.zip](https://github.com/howdyai/howdy-website/files/2973570/MAS1.3.1_Howdy.zip)
[MAS1.3.1_Scan mode.zip](https://github.com/howdyai/howdy-website/files/2973571/MAS1.3.1_Scan.mode.zip)
![MAS1 3 1_Howdy](https://user-images.githubusercontent.com/48272005/54471779-d26d3a00-47e4-11e9-9ee3-5c45b64dc770.png)
[MAS1.3.1_Pause button.zip](https://github.com/howdyai/howdy-website/files/2976660/MAS1.3.1_Pause.button.zip)

**Comment by benbrown** [comment]
@awalia13  Regarding the pause button, I believe this is happening because there are actually 2 buttons being swapped back and forth.  It is possible that I can change this so that the SAME BUTTON ELEMENT will be present (and focus will stay on that same button), but the text label will change.

Is there something additonal I will need to do to trigger the screen reader to read the label again? 

Regarding the logo link - I am not sure how to fix this! The link itself has a title tag -- however, it appears the screen reader is treating the SVG elements in a strange way.  What would you recommend?

**Comment by benbrown** [comment]
add aria-hidden to svg elements

aria-label can be switched on toggle

**Comment by benbrown** [comment]
OK! Aria-hidden has been added to the logo svg elements.  Hopefully it will read correctly now.

The pause toggle has been updated with an aria-live tag, and the text now updates inside the button rather than replacing the button entirely.

**Comment by v-gasikk** [comment]
Verified. Issue is Not Fixed. Hence Re-activating the bug.

**Comment by v-gasikk** [comment]
Please add aria-label for Howdy link, then screen reader will read correct information when focus is on "Howdy" link.
<a title="Back to dashboard" aria-label="Howdy home page back to dashboard" href="/dashboard">

**Comment by v-gasikk** [comment]
<a title="Back to dashboard" aria-label="Howdy home page back to dashboard" href="/dashboard">

**Comment by v-gasikk** [comment]
In achor tag add title="Back to dashboard" aria-label="Howdy home page back to dashboard" href="/dashboard"

**Comment by benbrown** [comment]
Done, please re-verify.

**Comment by awalia13** [comment]
Closed as it is working fine.
